src/Task01.js

```diff
@@ -8,9 +9,11 @@ const Task01 = () => {
 
     return (
         <section>
-            <h1>Task01</h1>
-            <LoginForm tryAuth={ tryAuth }/>
-        </section>
+                <h1>Task01</h1>
+                <CatchError>
```

Comment:
Przechwytywanie błędów nie działa w momencie, gdy błąd jest rzucany w zdarzeniu.
Trzeba to przepisać w taki sposób, aby błąd był rzucany poza eventem (można wykorzystać `state` o nazwie `error`)
https://pl.reactjs.org/docs/error-boundaries.html (żółta ramka)